Formula/nghttp2.rb

```diff
@@ -45,6 +48,13 @@ def install
     # https://github.com/macports/macports-ports/commit/54d83cca9fc0f2ed6d3f873282b6dd3198635891
     inreplace "src/shrpx_client_handler.cc", "return dconn;", "return std::move(dconn);"
 
+    # Don't build nghttp2 library - use the previously built one.
+    inreplace "Makefile.in", /(SUBDIRS =) lib/, "\\1"
+    inreplace Dir["**/Makefile.in"] do |s|
+      s.gsub!(%r{^(LDADD = )\$[({]top_builddir[)}]/lib/libnghttp2\.la}, "\\1-lnghttp2", false)
+      s.gsub!(%r{\$[({]top_builddir[)}]/lib/libnghttp2\.la}, "", false)
```

Comment:
Why don't we want `audit_result = true` here?

Reply:
I'm doing a glob over all Makefiles and some won't have this.

Reply:
That seems worth documenting in a comment.

Reply:
Ok. Don't really want to start another build unless I need to but I can do it on merge.

Reply:
Yup, no need to restart the build for it.

Reply:
The pattern I've seen used for this (that I like) is doing `if s.include?` here

Reply:
I'm not entirely sure what extra safety that adds?

Reply:
No safety but feels more self-documenting IMO.

Reply:
Yeh, it'd need some work but may be worth doing eventually. Also worth thinking about this being possible for e.g. using a formula's url/sha in a resource.